### ISSUE TITLE
More detail / examples on caching

### ISSUE THREAD

**Issue by cholmes** [opengeospatial/WFS_FES#38]
The section on caching is pretty light, https://rawgit.com/opengeospatial/WFS_FES/master/docs/17-069.html#_web_caching but is quite important.

There's a todo there for adding an OpenAPI example, which would be a good start. Opening this issue as it'd be really good to get that todo in there, and start figuring out how to test for caching. Should make at least recommendations on what headers to use (etags vs cache-control), when to update them / how long to set them for, proper http responses (304 not modified), making proper requests (If-Modified-Since or If-None-Match), etc. 

I think many people in the geospatial world aren't fluent in the web caching paradigm, so giving some solid recommendations I think would go a long way.

Ideally we'd also have a tutorial of some sort, showing how you could use a polling sort of mechanism with cache headers to keep two services in sync. And/or perhaps we make a little extension to request all features that are updated since a certain time.

**Comment by cportele** [comment]
I agree, I had feedback, too, that more guidance on caching and its role in the web architecture would be good. We need to see what needs to go into the standard (requirements, recommendations, permissions) and what into the informative guide (that was discussed today in the web-meeting).

We can add this also later in the process, but it is good to have this issue so we can keep this on our radar.

**Comment by cmheazel** [comment]
@cholmes This would be useful information but the Guide is the appropriate place to capture it.  If you would provide me with a bit of content, I'll create a new section.  Just enough to capture the key concepts will do.
